resources/qml/InviteDialog.qml

```diff
@@ -0,0 +1,224 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import im.nheko 1.0
+
+ApplicationWindow {
+    id: inviteDialogRoot
+
+    property string roomId
+    property string roomName
+    property InviteesModel invitees
+
+    function addInvite() {
+        if (inviteeEntry.text.match("@.+?:.{3,}")) {
+            invitees.addUser(inviteeEntry.text);
+            inviteeEntry.clear();
+        } else {
+            warningLabel.show();
+        }
+    }
+
+    function cleanUpAndClose() {
+        if (inviteeEntry.text !== "")
+            addInvite();
+        invitees.accept();
+        close();
+    }
+
+    title: qsTr("Invite users to ") + roomName
+    x: MainWindow.x + (MainWindow.width / 2) - (width / 2)
+    y: MainWindow.y + (MainWindow.height / 2) - (height / 2)
+    height: 380
+    width: 340
+
+    Shortcut {
+        sequence: "Ctrl+Enter"
+        onActivated: cleanUpAndClose()
+    }
+
+    Shortcut {
+        sequence: StandardKey.Cancel
+        onActivated: inviteDialogRoot.close()
+    }
+
+    ColumnLayout {
+        anchors.fill: parent
+        anchors.margins: 10
```

Comment:
Can you try using the standard spaings, like Nheko.paddingMedium? That way we don't have 2px different paddings everywhere :D

Reply:
There is still a margin: 10 here .-.

Reply:
Phooey! I'll get it later, busy rn